Externals/ode/ode/src/collision_quadtreespace.cpp

```diff
@@ -347,7 +347,7 @@ struct dxQuadTreeSpace : public dxSpace{
 };
 
 dxQuadTreeSpace::dxQuadTreeSpace(dSpaceID _space, dVector3 Center, dVector3 Extents, int Depth) : dxSpace(_space){
-	type = dQuadTreeSpaceClass;
+    type = dClassNumber::dQuadTreeSpaceClass;
```

Comment:
ODE uses tabs everywhere, please don't replace them with spaces.
```suggestion
	type = dClassNumber::dQuadTreeSpaceClass;
```